kpatch-build/kpatch-build

```diff
@@ -274,7 +274,13 @@ while [[ $# -gt 0 ]]; do
 	shift
 done
 
-TEMPDIR="$(mktemp -d /tmp/kpatch-build-XXXXXX)" || die "mktemp failed"
+# ensure cachedir and tempdir are setup properly
+mkdir -p "$CACHEDIR" || die "Couldn't create $CACHEDIR"
+mkdir -p "$TEMPDIR" && rm -rf $TEMPDIR/* || true
+
+# remove old LOGFILE
+rm -f $LOGFILE
```

Comment:
A few nits:
- it should die if `mkdir -p $TEMPDIR` fails
- no need to `mkdir -p $CACHEDIR` since `mkdir -p $TEMPDIR` will do that
- similarly, no need to `rm -f $LOGFILE` since `rm -rf $TEMPDIR/*` will do that

So I think this can be reduced to:
```
mkdir -p "$TEMPDIR" || die "Couldn't create $TEMPDIR"
rm -rf "$TEMPDIR"/*
```